docs/users/how_tos/modifiers_setup.md

```diff
@@ -0,0 +1,93 @@
+---
+id: modifiers_setup
+title: Setup configuration modifiers
+---
+
+To start using configuration modifiers, all you need to do is call
+`set_cfg_constructor` from a `PACKAGE` file.
+
+Start by adding the following to your root `PACKAGE` file:
+
+```python
+load("@prelude//cfg/modifier:cfg_constructor.bzl", "cfg_constructor_post_constraint_analysis", "cfg_constructor_pre_constraint_analysis")
+load("@prelude//cfg/modifier:common.bzl", "MODIFIER_METADATA_KEY")
+
+native.set_cfg_constructor(
+    stage0 = cfg_constructor_pre_constraint_analysis,
+    stage1 = cfg_constructor_post_constraint_analysis,
+    key = MODIFIER_METADATA_KEY,
+    aliases = struct(),
+    extra_data = struct(),
+)
+```
+
+`set_cfg_constructor` is a Buck2 builtin used to setup configuration
+modifiers. It supports a few configuration points:
+
+- `stage0` and `stage1` are used to resolve modifiers from the [target
+  platform](../../rule_authors/configurations) (which is now considered
+  deprecated) and the modifiers set in the nearest `PACKAGE` file, on
+  the target, and on the CLI. You _can_ override those, but the
+  functions provided by the prelude basically do what you would expect.
```

Comment:
Would it make sense for these to be populated by default?

Reply:
I don't see why not, and it should be trivial to set default values. I don't think there has been a lot of thought around ergonomics here yet.

Reply:
Resolving as this is out of scope for this PR. Feel free to open an issue or a discussion for Meta about this.